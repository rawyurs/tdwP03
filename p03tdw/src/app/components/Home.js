export default function Home() {
  return (
    <>
      <h2 id="criado">Titulo Novo</h2>
      <p>Teste</p>
      <p>umbigo</p>
      <p>piscina</p>
      <p>amizade2</p>
      <h1>Hello World!</h1>
      <p>Ricardo</p>
    </>
  );
}
