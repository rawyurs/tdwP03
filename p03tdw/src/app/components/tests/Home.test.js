import { render, screen } from "@testing-library/react";
import Home from "../Home";
import React from "react";
import "@testing-library/jest-dom";

test("renders Home component", () => {
  const { asFragment } = render(<Home />);

  // Verifique se o título "Hello World!" está no documento
  const headingElement = screen.getByText(/umbigo/i);
  expect(headingElement).toBeInTheDocument();

  // Adicionando o teste de Snapshot
  expect(asFragment()).toMatchSnapshot();
});
