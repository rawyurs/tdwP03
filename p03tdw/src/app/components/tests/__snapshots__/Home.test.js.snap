// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component 1`] = `
<DocumentFragment>
  <h2
    id="criado"
  >
    Titulo Novo
  </h2>
  <p>
    Teste
  </p>
  <p>
    umbigo
  </p>
  <p>
    piscina
  </p>
  <p>
    amizade2
  </p>
  <h1>
    Hello World!
  </h1>
  <p>
    Ricardo
  </p>
</DocumentFragment>
`;
